rust/libnewsboat/src/filterparser.rs

```diff
@@ -270,7 +270,7 @@ pub fn parse(expr: &str) -> Result<Expression, String> {
                     &gettext("Parse error at position %{}: expected %s"),
                     PRIu64,
                     pos as u64,
-                    expected
+                    &gettext(expected)
```

Comment:
When I run `make extract`, I don't see any of the messages being added.
Is there something else I should run instead of `make extract`?

Reply:
Ah, no, that's my bad — this doesn't work because `xtr` can't know what `expected` is going to be. And I can't move `gettext()` calls to where `EXPECTED_*` statics are defined because this function can't be run (and doesn't make sense) during compile-time.

I'll think what can be done about that. Thanks for spotting the problem!

Reply:
I fixed it, and `make extract` now updates the POT file accordingly. The solution is rather ugly because `nom::context` macro requires a `&'static str`, so I can't use any of the usual tricks to avoid runtime checks (e.g. I could use an `enum`, then `expected_to_i18n_msg` would use `match` to guarantee that all possible errors are converted). I'll go file an issue with Nom crate regarding this.

Reply:
Filed https://github.com/Geal/nom/issues/1197.

Reply:
Thank you for fixing it